tests/zfs-over-legacy.nix

```diff
@@ -1,7 +1,8 @@
 { pkgs ? (import <nixpkgs> { })
-, makeDiskoTest ? (pkgs.callPackage ./lib.nix { }).makeDiskoTest
+, makeDiskoTest ? (pkgs.callPackage ../lib { }).testLib.makeDiskoTest
 }:
 makeDiskoTest {
+  inherit pkgs;
```

Comment:
We did we need passing pkgs here instead of passing it in callPackage already?

Reply:
lib can't depend on pkgs, because of inifinite recursion